From 7b975daca24b580efbbcf235e1500f32bbdb1fe6 Mon Sep 17 00:00:00 2001
From: Tong Zhang <zhangt@frib.msu.edu>
Date: Fri, 15 Dec 2017 16:05:39 -0500
Subject: [PATCH 1/2] Import flame_utils into python-flame.

---
 python/flame/__init__.py | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/python/flame/__init__.py b/python/flame/__init__.py
index 58673c8..c6197a9 100644
--- a/python/flame/__init__.py
+++ b/python/flame/__init__.py
@@ -48,4 +48,9 @@ __all__ = ['Machine',
     'GLPSParser',
 ]
 
+try:
+    from flame_utils import *
+except:
+    pass
+
 __version__ = '1.4.0'
-- 
2.7.4

