From 9e6ff3355e311c4bd2ab8e3382a5a6530f505dbe Mon Sep 17 00:00:00 2001
From: Tong Zhang <zhangt@frib.msu.edu>
Date: Fri, 15 Dec 2017 16:07:14 -0500
Subject: [PATCH 2/2] Patch version to 1.4.1.

---
 python/flame/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/flame/__init__.py b/python/flame/__init__.py
index c6197a9..f1a85d2 100644
--- a/python/flame/__init__.py
+++ b/python/flame/__init__.py
@@ -53,4 +53,4 @@ try:
 except:
     pass
 
-__version__ = '1.4.0'
+__version__ = '1.4.1'
-- 
2.7.4

